README.md

```diff
@@ -8,20 +8,25 @@
 
 Implementation of Repository pattern for Laravel (on top of Eloquent)
 
-## Laravel 5.x
+## Laravel 5.5
 
 Install the ```saritasa/laravel-repositories``` package:
 
 ```bash
 $ composer require saritasa/laravel-repositories
 ```
 
-*Note:* For Laravel 5.4 use 1.* versions, for Laravel 5.5+ use 2.*
-
 ## Available classes
 
-### IRepository
-Interface, declaring common methods
+### RepositoryFactory
+Base iRepositoryFactory implementation. You can register IRepository implementation to some model. In when you will be getting repository for this model - you will receive this registered object.
+
+**Example**
+```php 
+    $repositoryFactory = app(IRepositoryFactory::class);
+    $repositoryFactory->register(User::class, Repository::class);
+```
+*Your repository must implement IRepository contract and receive string $modelClass in constructor.*
```

Comment:
_... and accept managed model class name in constructor_, IMHO, no need to point to implementation ($modelClass)